deployment/ccip/changeset/testhelpers/test_helpers.go

```diff
@@ -921,16 +1061,35 @@ func setTokenPoolCounterPart(chain deployment.Chain, tokenPool *burn_mint_token_
 		return err
 	}
 
-	tx, err = tokenPool.AddRemotePool(
-		actor,
-		destChainSelector,
-		destTokenPoolAddress.Bytes(),
-	)
+	supported, err := tokenPool.IsSupportedChain(&bind.CallOpts{}, destChainSelector)
 	if err != nil {
-		return fmt.Errorf("failed to set remote pool on token pool %s: %w", tokenPool.Address(), err)
+		return err
+	}
+	if !supported {
+		return fmt.Errorf("token pool %s is not supported on chain %d", tokenPool.Address(), destChainSelector)
 	}
 
-	_, err = chain.Confirm(tx)
+	fmt.Println("tokenPool", tokenPool.Address(), "supported", supported)
+	fmt.Println("destTokenPoolAddress", destTokenPoolAddress)
+	fmt.Println("destChainSelector", destChainSelector)
+
+	remotePools, err := tokenPool.GetRemotePools(&bind.CallOpts{}, destChainSelector)
+	if err != nil {
+		return err
+	}
+	fmt.Println("remotePools", remotePools)
+
+	// tx, err = tokenPool.AddRemotePool(
+	// 	actor,
+	// 	destChainSelector,
+	// 	destTokenPoolAddress,
+	// )
+	// if err != nil {
+	// 	return fmt.Errorf("failed to set remote pool on token pool %s: %w", tokenPool.Address(), err)
+	// }
+
+	// _, err = chain.Confirm(tx)
+	// return err
```

Comment:
This seems unfinished and will be a regression on EVM

Reply:
Can we split out `DeployTransferrable` changes and just get the fee quoter/offramp changes merged?